roles/pulp_common/templates/settings.py.j2

```diff
@@ -1,3 +1,5 @@
+# Do NOT edit this file. It may be subject of change when PULP is updated.
+# Use {{ pulp_settings_file_local }} instead.
```

Comment:
dynaconf merges the 2 settings files, correct?

How about "put new or overridden settings in {{ pulp_settings_file_local }} instead."

Reply:
It does and makes *local* file having a priority.